Content.Server/GameTicking/Rules/TraitorRuleSystem.cs

```diff
@@ -240,4 +264,30 @@ private string GenerateBriefing(string[]? codewords, Note[]? uplinkCode, string?
 
         return traitors;
     }
+
+    // SS220 Dynamics begin
+    private void InitDynamic(EntityUid ent, TraitorRuleComponent? rule)
```

Comment:
```suggestion
    private void InitDynamic(EntityUid uid, TraitorRuleComponent? rule)
```
Или привести к виду Entity<T>